File: src/main/java/id/hackathonmerdeka/hackmdk3/controller/IndexController.java
package id.hackathonmerdeka.hackmdk3.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author Arthur Purnama (arthur@purnama.de)
 */
@Controller
public class IndexController {

    @RequestMapping(value = "/{path:^(?!api)(?!index)(?!bower)(?!css)(?!error)(?!images)(?!js)(?!templates).*$}/**")
    public String forward(@PathVariable(value = "path") String path) {
        return "forward:/";
    }
}

File: src/main/resources/static/js/controller/IndexController.js
/**
 * @author Arthur Purnama (arthur@purnama.de)
 */
birokrazyApp.controller('IndexController', ['$scope', 'IndexService', function ($scope, indexService) {
    indexService.findAll().then(function(data) {
        $scope.highlights = data;
    });
}]);